src/DoctrineObject.php

```diff
@@ -52,6 +58,7 @@ public function __construct(ObjectManager $objectManager, $byValue = true)
     {
         $this->objectManager = $objectManager;
         $this->byValue = (bool) $byValue;
+        $this->inflector = InflectorFactory::create()->build();
```

Comment:
Or should be injected?

Reply:
For each of these instances do not inject the inflector but provide a getInflector and setInflector method.  I don't want to change the constructor but we need to give the user the ability to change the inflector and creating an object then setting the inflector is a fine option since changing the inflector would be a rare occurance.

Reply:
Use the setter method in the constructor.

Reply:
What if I change constructor by add optional parameter with inflector? If null then use default created by factory, like now.